### PR TITLE
Project 2 spec

### PR DESCRIPTION
**Pull request by deonn1**
Still need to update the links on the webpage and connect it to the home page.

### CODE REVIEW

**Review by ieee-uci — DISMISSED** [review]
Suggested changes left on discord -Antonio

**Review by bcbielecki — DISMISSED** [review]
Hi guys! Please update the PR with all the Project 2 doc changes from the last month. Let me know when you're done by resolving my change request and requesting another review.

**Review by bcbielecki — CHANGES_REQUESTED** [review]
Hi guys! Good work on the page. I left a couple of comments, which should be quick to resolve.

This next one may take a bit longer: I noticed that the images don't properly break down when the viewport decreases in width. The text and images overflow (via scrollbar) off the right edge of the viewport.

Instead, we should see the images with any alignment break down into a "center-aligned"-"break text"-style.

This can be accomplished using Bootstrap breakpoints along with bootstrap's CSS classes. You can find more on this here: https://getbootstrap.com/docs/5.1/layout/breakpoints/

Talk to Dylan and Gavin about this. They will be working through a similar issue on their project page.

File: project_2.html [hunk 1]
```diff
@@ -0,0 +1,603 @@
...
+					project, the IC package is DIP.
+					<br>
+					Note that the organization of the pinout differs from the schematics below. The true location of the ICs pins is shown by the 
+					pinout, not the schematic. Use the notch on a DIP-style IC to find its correct orientation.	
+					<br><br>	
+			</p>
+
+			<h4 class="mb-3 fw-light" id="pj-heading-NE555PIC" style="display: inline;">How It Works</h4>	
+			<p class="mb-5">
+				Learn more about 555 Timer’s functionality in the lecture material <span class="text-blue">here [Webpage Link]</span>.
```
Comment (bcbielecki) [inline project_2.html:267]: image

Please replace this placeholder with a link to the lectures page.

We could rewrite this line to say "Learn more about the 555 Timer's functionality in the Lecture II material here"

File: project_2.html [hunk 2]
```diff
@@ -0,0 +1,603 @@
...
+				KiCAD (as seen below). If you are not enrolled in the course, you may alternatively use a perfboard.
+				<br>
+				<img class="img-fluid my-3 me-5 float-start" style="max-width: 700px;"
+					src="assets/images/visuals/ops_kicad_example.png"> 
+				</p>
+
+			<p class="mb-5">
+				Soldering to a PCB is much like soldering to a protoboard, and you may even find it easier. 
+				For more soldering tips visit our workshop material <a class="link-primary" 
+				href="https://docs.google.com/presentation/d/11AWxHFSlhvF4bWnvbFpBkxV5KOSYh7OgkUuCPrOEb2k/edit?usp=sharing" target="_blank" rel="noopener noreferrer">here</a>.
```
Comment (bcbielecki) [inline project_2.html:297]: image

Now that there's a workshops page, please update the hyperlink to redirect to workshops.html

File: project_2.html [hunk 3]
```diff
@@ -0,0 +1,603 @@
...
+							<img src='assets/images/visuals/ops_schem_proj2a.png' class="pj-schem">
+						</div>
+					</div>
+				</div>
+
+				<div class="accordion-item">
+					<h2 class="accordion-header">
+						<button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
+							data-bs-target="#schemCollapse2" aria-expanded="true" aria-controls="schemCollapse2">
+							Schematic B (555 Piano)
```
Comment (bcbielecki) [inline project_2.html:502]: image
image

Please update the Schematic B to be a more legible sizing (consistent with Schematic A)

**Review by bcbielecki — CHANGES_REQUESTED** [review]
Hi guys! Here are my last couple review comments.

Please also update the ops_header.js file to include the new project 2 link in the "Projects" dropdown

**Pull request closed without merging**